src/helpers.js

```diff
@@ -353,15 +355,55 @@ module.exports = {
         resolve(hash);
       });
     })
-
-
   },
+
   comparePassword: (myPlaintextPassword, hash) => {
     return new Promise((resolve, reject) => {
       bcrypt.compare(myPlaintextPassword, hash, function (err, result) {
         if (err) reject(err);
         resolve(result);
       });
     });
+  },
+
+  checkPersonName(personName){
+    personName = this.checkString(personName, "personName");
+
+    let numSpaces = 0;
+    for(let i = 0; i < personName.length; i++){
+      if(personName[i] === ' '){
+        numSpaces = numSpaces + 1;
+      }
+    }
+
+    if(numSpaces !== 1){
+      throw "Error: incorrect format for name";
+    }
+
+    let name = personName.split(' ');
+    let firstName = name[0];
+    let lastName = name[1];
+
+    firstName = this.checkName(firstName);
+    lastName = this.checkName(lastName);
+
+    let fullName = firstName + " " + lastName
+
+    return fullName;
+  }, 
+
+  checkRating(rating){
+    if(!rating){
+      throw "Error: rating must be provided";
+    }
+    if(typeof rating !== "number"){
```

Comment:
All input from the user will be type string. So, I think before you check that rating is of type number, you should do Number(rating), and then check if it is not type NaN